### PR TITLE
Un-inline SVGs with CSP-violating features

### PR DESCRIPTION
**Pull request by janbrasna**
## One-line summary

SVGs with inline styling that can't be easily turned into attributes (think groups etc.) can't be used/included inline in documents because that would violate non-unsafe-inine CSP policy. This moves them out to media for img linking.

## Significant changes and points to review

Some occurrences were for WNP119-124 that were removed recently, so these could be cleaned up.

BTW welcome/9 images were kinda broken in a few webkits anyway, pretty much some of the advanced svg features didn't work out when inline perhaps, so this fixes that along with it.

## Issue / Bugzilla link

#14840 (see https://github.com/mozilla/bedrock/pull/14841#pullrequestreview-2182272728)

## Testing

http://localhost:8000/en-US/firefox/welcome/11/
http://localhost:8000/en-US/firefox/welcome/10/
http://localhost:8000/de/firefox/welcome/9/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/welcome/page10.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h1 class="c-main-title">{{ ftl('welcome-page10-main-heading') }}</h1>
 
       <div class="mzp-c-picto">
         <div class="mzp-c-picto-image">
-          {% include 'firefox/welcome/includes/page10/laptop.svg' %}
+          <img src="{{ static('img/firefox/welcome/page10/laptop.svg') }}" loading="lazy" alt="">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/welcome/page10.html:48]: **nit:** `loading="lazy"` should come before `src` (see https://bugzilla.mozilla.org/show_bug.cgi?id=1647077)

Same with other occurrences below.

File: bedrock/firefox/templates/firefox/welcome/page11.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <h1 class="c-main-title">{{ ftl('welcome-page11-main-heading') }}</h1>
 
       <div class="mzp-c-picto">
         <div class="mzp-c-picto-image">
-          {% include 'firefox/welcome/includes/page11/globe.svg' %}
+          <img src="{{ static('img/firefox/welcome/page11/globe.svg') }}" alt="">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/welcome/page11.html:41]: **issue:** It looks like this page supports both light and dark theme, so when viewed in dark mode it's hard to see the individual image lines. I'd suggest changing these to use `<picture>`, which can then serve light / dark versions of each image when needed using `(prefers-color-scheme: light/dark)`. You can see the page in prod for reference: https://www.mozilla.org/de/firefox/welcome/11/

**Review by alexgibson — CHANGES_REQUESTED** [review]
Thanks for this, it will be a big help in blocking the CSP change 🎉 

A couple of small issues below but nothing major. Let me know if you need a hand at all creating assets for the light/dark mode images.

File: bedrock/firefox/templates/firefox/welcome/page11.html (hunk 2 above)
Comment (janbrasna) [reply bedrock/firefox/templates/firefox/welcome/page11.html:41]: Holy *! I've checked it out and thought it was as designed, but now see the `invert-fill` class for it from the document level. I'll need to think a bit about this, as I usually just "hue-rotate" the hell out of it via CSS but this is def more involved. Very smart execution though.

Will explore using `fill="currentColor"` for the blacks and setting the dark mode appearance via color on the DOM parent, will see how that goes… Thanks for catching this!

File: bedrock/firefox/templates/firefox/welcome/page10.html (hunk 1 above)
Comment (janbrasna) [reply bedrock/firefox/templates/firefox/welcome/page10.html:48]: Learning something new every day ([MDN: loading#usage_notes](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading#usage_notes)) — and in this case of welcomes it's the only engine relevant so, silly me;) I may remove them after all, as they're right around the edge of the fold for various screen sizes, so they may not add any improvement whatsoever…

Q: aren't welcome pages actually downloaded in the background, as per https://support.mozilla.org/kb/firefox-browser-welcome-pages#w_how-do-firefox-browser-welcome-pages-get-delivered? Or is it meant that only their list is downloaded ahead, and the actual rendered pages chosen to display and their assets still load "live", online?

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/welcome/page10.html:48]: I believe it's just the list that gets downloaded 👍

File: bedrock/firefox/templates/firefox/welcome/page11.html (hunk 2 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/welcome/page11.html:41]: Any alternative solutions are welcome here fwiw. I believe this page has probably seen _most_ of its traffic, so it's only worth making the effort to migrate the images as they were if it's not too much effort.

File: bedrock/firefox/templates/firefox/welcome/page10.html (hunk 1 above)
Comment (janbrasna) [reply bedrock/firefox/templates/firefox/welcome/page10.html]: In that case I'm leaving it out completely, to rather load the images eagerly, not risking any potential perception of the pages as being "slow" if something gets loaded delayed…

File: bedrock/firefox/templates/firefox/welcome/page11.html (hunk 2 above)
Comment (janbrasna) [reply bedrock/firefox/templates/firefox/welcome/page11.html:41]: This one was fun;) Of course that `currentColor` also has the usual document boundary limit so it won't inherit the colour from a surrounding DOM it's not a part of…:/

But luckily, the classes and media prefers were already there, just needed moving inside the SVGs 😇

**Review by alexgibson — APPROVED** [review]
r+ thanks!

File: bedrock/firefox/templates/firefox/welcome/page11.html (hunk 2 above)
Comment (janbrasna) [reply bedrock/firefox/templates/firefox/welcome/page11.html:41]: FYI this has a caveat with some Safari versions not applying the correct color-scheme to child documents like here (i.e. the actual SVG opened just itself as a top document follows light/dark; but loaded inside a page that prefers light/dark won't switch the atmedia rule:/…)

But given this is an in-product page, I assumed Gecko compatibility is sufficient.

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/welcome/page11.html:41]: Yeah i think we're OK here seeing as this is a Firefox only page. Typically we'd use a `<picture>` element here instead, but what you have works for this use case. Thanks!

Comment (janbrasna) [reply bedrock/firefox/templates/firefox/welcome/page11.html:41]: Never realised there would be so much quirks even today. _Learning something new every day along the way… ;)_

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/welcome/page11.html:41]: This gotcha is a new one for me too fwiw, TIL!

**Pull request merged**